docs/components/PLACEHOLDERS.md

```diff
@@ -204,7 +204,7 @@ Then, let's create a new localization key for each of your supported languages:
 
 ```json
 {
-  "o3r-increment-key": "Cela fait {increment, plural, =1 {1 seconde}} other {{{increment} secondes} que tu as ouvert cette page"
+  "o3r-increment-key": "Cela fait {increment, plural, =1 {1 seconde}} other {{{increment} secondes}} que tu as ouvert cette page"
```

Comment:
```suggestion
  "o3r-increment-key": "Cela fait {increment, plural, =1 {1 seconde} other {{{increment}} secondes}} que tu as ouvert cette page"
```
not 100% sure...

Reply:
Honestly I just followed the other example without thinking much just to unb unblock the ci.
I will trust you on that one.